mesonbuild/modules/gnome.py

```diff
@@ -784,8 +784,8 @@ def _devenv_prepend(self, varname: str, value: str) -> None:
             self.devenv = build.EnvironmentVariables()
         self.devenv.prepend(varname, [value])
 
-    def get_devenv(self) -> T.Optional[build.EnvironmentVariables]:
-        return self.devenv
+    def postconf_hook(self, b: build.Build) -> None:
+        b.devenv.append(self.devenv)
```

Comment:
`self.devenv` can be None, it should not be added to list in that case.

Reply:
Oops. For the install scripts I didn't bother to check this since I simply used a list that could be empty, but for a devenv we need exactly zero or one and then need to repeatedly modify it...

Will fix.